### DISCUSSION TITLE
[Yet Another Broken Update] In Fedora 36, new update of gns3server was released without fixing gns3gui's python dependency issue.

### DISCUSSION THREAD

**Discussion by PranavBhattarai** [GNS3/gns3-gui#3352] [Q&A]
I don't want to pollute the GitHub issues especially when the problem is not unique at this point, I just wanted to grab attention toward a problem so a fix is released.
```
Problem: cannot install the best update candidate for package gns3-gui-2.2.32-1.fc36.noarch
  - nothing provides python3.10dist(distro) >= 1.7 needed by gns3-gui-2.2.33.1-1.fc36.noarch
=======================================================================================================================================
 Package                        Architecture                 Version                               Repository                     Size
=======================================================================================================================================
Skipping packages with broken dependencies:
 gns3-gui                       noarch                       2.2.33.1-1.fc36                       updates                       4.6 M
...
 ```
 
 ```
  ⟹  dnf repoquery --installed | grep -i gns3
gns3-gui-0:2.2.32-1.fc36.noarch
gns3-net-converter-0:1.3.0-20.fc36.noarch
gns3-server-0:2.2.33.1-1.fc36.noarch
```

**Comment by eantowne** [comment]
I do not believe that the Fedora repo provided version of GNS3 is provided by the actual GNS3 team. Please contact the package provider regarding this issue. Package owner can be found here: https://fedoraproject.org/wiki/Changes/GNS3.


2.2.33.1 install fine on Ubuntu via official package provided by GNS3 team via PPA.

**Reply by PranavBhattarai** [reply]
Thank you. I will reach them about it.

**Reply by eantowne** [reply]
Alternatively, you can try to install via pip, as the developers also make the packages available via that distribution channel as well. More in formation can be found at https://docs.gns3.com/docs/

**Reply by PranavBhattarai** [reply]
https://bugzilla.redhat.com/show_bug.cgi?id=2104903

The fix is on the way... soon..
